ci/prow/config_knative.yaml

```diff
@@ -127,6 +127,14 @@ periodics:
       full-command: "./test/e2e-tests.sh --istio-version 1.1.7 --mesh"
     - custom-job: istio-1.1.7-no-mesh
       full-command: "./test/e2e-tests.sh --istio-version 1.1.7 --no-mesh"
+    - custom-job: k8s-1.12-istio-1.1
+      full-command: "./test/e2e-tests.sh --cluster-version \"1.12\" --istio-version 1.1-latest"
```

Comment:
You can drop the escaped quotes for the cluster version, it will increase readability. Here and everywhere.

Reply:
If `isNum` returns true for "./test/e2e-tests.sh --cluster-version \"1.12\" --istio-version 1.1-latest", it's buggy.

Reply:
It returns true for "1.12"

Reply:
Ah. It's because of https://github.com/knative/test-infra/blob/master/ci/prow/make_config.go#L523

This is tricky. I'm not sure if single quotes would work here, but definitely add a comment about the need of quoting the value.

Reply:
Single quote doesn't work as is, it would work if change this line to also return if contains single quote: https://github.com/knative/test-infra/blob/d8e74c06fd5a0d54fe246d8c51329b3b103ba135/ci/prow/make_config.go#L907

I can try to make it work

Reply:
done, please take a look